src/main/java/io/github/thebusybiscuit/slimefun4/implementation/guide/SurvivalSlimefunGuide.java

```diff
@@ -622,12 +648,32 @@ private static ItemStack getDisplayItem(Player p, boolean isSlimefunRecipe, Item
         if (isSlimefunRecipe) {
             SlimefunItem slimefunItem = SlimefunItem.getByItem(item);
 
-            if (slimefunItem == null) {
+            if (slimefunItem == null || slimefunItem.canUse(p, false)) {
+                // Return if the item is usable or isn't a Slimefun item
                 return item;
             }
 
-            String lore = Slimefun.hasPermission(p, slimefunItem, false) ? "&fNeeds to be unlocked elsewhere" : "&fNo Permission";
-            return slimefunItem.canUse(p, false) ? item : new CustomItem(Material.BARRIER, ItemUtils.getItemName(item), "&4&l" + SlimefunPlugin.getLocalization().getMessage(p, "guide.locked"), "", lore);
+            if (!SlimefunPlugin.getPermissionsService().hasPermission(p, slimefunItem)) {
+                return new CustomItem(Material.BARRIER, ItemUtils.getItemName(item), "&4&l" + SlimefunPlugin.getLocalization().getMessage(p, "guide.locked"), "", "&fNo Permission");
+            } else {
+                ItemStack result = slimefunItem.getItem().clone();
```

Comment:
Is the name the only thing left over from this clone? I feel it'd be better to just create a new item instead of cloning + changing 99% of it.

Reply:
The slimefun item ID and custom model data if theres one will be preserved as well